web/src/features/charts/bar-breakdown/BarBreakdownChart.tsx

```diff
@@ -115,7 +123,16 @@ function BarBreakdownChart({
       <ChartTitle
         titleText={titleText}
         unit={graphUnit}
-        isEstimated={hasEstimationPill}
+        badge={
+          hasEstimationPill ? (
+            <EstimationBadge
+              text={pillText}
+              Icon={
+                estimationMethod === EstimationMethods.TSA ? CircleDashed : TrendingUpDown
```

Comment:
Could we add the `FORECASTS_HIERARCHY` method here as well (and other places we use preliminary) since it's also a TSA model enhanced with solar and wind forecasts?

Maybe it should be a follow up PR though.

Reply:
Good suggestion - if it's a quick change then I'll include it in this one!

Reply:
We use `EstimationMethods.TSA` in [other places in the code]( https://github.com/search?q=repo%3Aelectricitymaps%2Felectricitymaps-contrib+EstimationMethods.TSA&type=code) as well. I assume that we'll want to check for `FORECASTS_HIERARCHY` in these places too for consistency?

Reply:
Probably yeah. Not sure if they are all identical but if you want to change them in here I'll help go through them on a 1:1 basis.

Reply:
Ok, I assumed that all of the instances in the search that stored the results in a variable named something like `isTSA` or `tsaCount` should be omitted. LMK if that instinct is wrong.
Omitted instances:
- https://github.com/electricitymaps/electricitymaps-contrib/blob/c1b42b0ef7e6d02b074cb5f0d4c8131bdc9661d7/web/src/features/charts/graphUtils.ts#L218
- https://github.com/electricitymaps/electricitymaps-contrib/blob/c1b42b0ef7e6d02b074cb5f0d4c8131bdc9661d7/web/src/features/panels/zone/EstimationCard.tsx#L56

Reply:
Not sure it's wrong but I'm also not sure we are doing that correctly as is.

IMO Forecast Hierarchy should be considered TSAv2, under the hood it's still based on TSA with some specific forecast based estimation for solar and wind (for now).

So the question is for the estimation card and graph utils if we should consider TSA and Forecast Hierarchy to be the same thing?

I see that the estimation card asks for feedback if it's TSA based but IMO it should also do that for Forecast Hierarchy, maybe even display the TSA card itself or a variant of it as that uses the dashed circle Preliminary icon.

Reply:
I would be fine with merging this as is now and then do some follow up cleanup, having a specific card for TSA don't even make much sense IMO, not anymore.

Reply:
> IMO Forecast Hierarchy should be considered TSAv2

Ahh, that's helpful. I wasn't sure what the relationship was between the two